index.js

```diff
@@ -759,3 +759,28 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+	var negatedMethods = {};
+	Object.keys(source).forEach(function (key) {
+		var value;
+    var negatedMethod;
+		if ( source.hasOwnProperty(key) ) {
+			value = source[ key ];
+			if ( is.fn(value) ) {
+				negatedMethod = negatedMethods[ key ] = function () {
+					return !value.apply(this, arguments);
```

Comment:
i think you can use `null` instead of `this` here, since none of the functions should be relying on `this`.